### PR TITLE
Bratseth/send result on exception

### PR DESCRIPTION
**Pull request by bratseth**
@vekterli please review

No changes here, the thing I wanted to do turned out to not be necessary.

### CODE REVIEW

File: vespa-http-client/src/main/java/com/yahoo/vespa/http/client/FeedClient.java [hunk 1]
```diff
@@ -108,4 +84,29 @@ static void feedXml(InputStream inputStream, FeedClient feedClient, AtomicIntege
...
+        /**
+         * Called with an exception whenever an endpoint specific error occurs during feeding.
+         * The error may or may not be transient - the operation will in both cases be retried until it's successful.
+         * This callback is intended for application level monitoring (logging, metrics, altering etc).
+         * Document specific errors will be reported back through {@link #onCompletion(String, Result)}.
+         *
+         * @see FeedEndpointException
+         * @param exception An exception specifying endpoint and cause. See {@link FeedEndpointException} for details.
+         */
+        // TODO Vespa 7: Remove empty default implementation
```
Comment (vekterli) [inline vespa-http-client/src/main/java/com/yahoo/vespa/http/client/FeedClient.java]: This class is likely to already have been touched on the `7` branch

File: vespa-http-client/src/main/java/com/yahoo/vespa/http/client/Result.java [hunk 2]
```diff
@@ -83,10 +71,11 @@ public Object getContext() {
     public boolean isSuccess() {
         return success;
     }
+
     /**
      * @deprecated use resultType on items getDetails() to check  operations.
-     * Returns true if an error is transient, false if it is permanent. Irrelevant
-     * if {@link #isSuccess()} is true (returns true in those cases).
+     *             Returns true if an error is transient, false if it is permanent. Irrelevant
+     *             if {@link #isSuccess()} is true (returns true in those cases).
```
Comment (vekterli) [inline vespa-http-client/src/main/java/com/yahoo/vespa/http/client/Result.java]: This method is already removed on the `7` branch

**Review by vekterli — COMMENTED** [review]
As it currently stands, merging this branch will conflict with existing changes on the `7` branch. Suggest factoring out these changes into a separate PR for that branch. Otherwise LGTM

**Comment by bratseth** [comment]
Not worth the effort to merge this

**Pull request closed without merging**